tests/e2e/env/bin/docker-compose.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+if [[ $1 ]]; then
+#
+# Set environment variables for Docker
+#
+	if ! [[ $WP_VERSION =~ ^[0-9]+\.[0-9]+ ]]; then
+		WP_VERSION=$(git ls-remote --tags https://github.com/wordpress/wordpress.git | cut -f3 -d/ | grep -E '^\d+\.\d+(.\d+)*$' | tail -n 1)
+	fi
+	if [[ $WP_VERSION =~ ^[0-9]+\.[0-9]+ ]]; then
+		export WORDPRESS_VERSION=$WP_VERSION
+	else
+		export WORDPRESS_VERSION="5.4.2"
```

Comment:
WDYT to use `latest` here instead of the fixed version?

Reply:
@renatho If you create a docker container with `wordpress:latest` at version 5.2, docker caches that version as `latest`. When 5.3, .4, etc. are released the container WP version remains 5.2.

Reply:
Ah, I got! Thank you for the answer! :)